pkg/app/metadata.json

```diff
@@ -0,0 +1,963 @@
+{
+  "acl": {
+    "create": {
+      "examples": [{
+        "cmd": "fastly acl create --name robots --version latest",
+        "title": "Create a new ACL attached to the specified service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#create-acl"]
+    },
+    "delete": {
+      "examples": [{
+        "cmd": "fastly acl delete --name robots --version latest",
+        "title": "Delete an ACL from the specified service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#delete-acl"]
+    },
+    "describe": {
+      "examples": [{
+        "cmd": "fastly acl describe --name robots --version latest",
+        "title": "Retrieve a single ACL by name for the version and service"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#get-acl"]
+    },
+    "list": {
+      "examples": [{
+        "cmd": "fastly acl list --version latest",
+        "title": "List ACLs for the version and service"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#list-acls"]
+    },
+    "update": {
+      "examples": [{
+        "cmd": "fastly acl update --name robots --new-name blocklist --version latest",
+        "title": "Update an ACL for a particular service and version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#update-acl"]
+    }
+  },
+  "acl-entry": {
+    "create": {
+      "examples": [{
+        "cmd": "fastly acl-entry create --acl-id 50CCynNwD3BjaGJjMTFB1j --ip 64.18.0.0",
```

Comment:
Docs team maintains a list of example placeholder data: https://github.com/fastly/fastly-docs/blob/main/_doc/style-guides/standard-examples-and-placeholders.md. Prefer these if possible.